### ISSUE TITLE
Criar estrutura de pastas inicial para o plugin

### ISSUE THREAD

**Issue by daltonmatos** [sixcodes/vimspector-sensible-defaults#1]
Acho que por enquanto bastaria ter as pastas `doc/`, `plugin/` e `autoload/`.

@abnt713

**Comment by abnt713** [comment]
Certo, posso começar com essa parte. Por favor, me coloque como asignee :+1:

**Comment by daltonmatos** [comment]
Acho que vale, na pasta `doc/`, já deixar um `vinspector-sensible-defaults.txt` com uma introdução ao projeto.
Dessa forma quando fizermos a instalação dessa versão já vai dar pra fazer `:help vimspector-sensible-defaults`. 🎉🎉

**Comment by daltonmatos** [comment]
Aqui tem uma doc sobre o formato do helpfile: http://vimdoc.sourceforge.net/htmldoc/helphelp.html#help-writing

Mas podemos pegar qualquer helpfile de exemplo, como o do vim-lsp (https://github.com/prabirshrestha/vim-lsp/blob/master/doc/vim-lsp.txt)
